schemas/schemas.py

```diff
@@ -220,6 +220,10 @@ class _BaseMapping(BaseModel):
     """
     Hyperlinks to web locations that provide more information about the mapping.
     """
+    extra: dict[NonEmptyString, Any] | None = None
```

Comment:
Any other suggestions for `extra` here? It's a loaded term in Python packaging, often used to refer to `optional-dependencies`.